web_src/js/features/repo-issue.js

```diff
@@ -638,8 +638,9 @@ export function initRepoIssueTitleEdit() {
         }
       }
       if (prTargetUpdateUrl) {
-        const newTargetBranch = document.querySelector('#pull-target-branch').getAttribute('data-branch');
-        const oldTargetBranch = document.querySelector('#branch_target').textContent;
+        // merged pr can not edit target branch, so they maybe null
+        const newTargetBranch = document.querySelector('#pull-target-branch')?.getAttribute('data-branch');
+        const oldTargetBranch = document.querySelector('#branch_target')?.textContent;
```

Comment:
Thank you for catching the bug, I missed a case.

But I think we shouldn't use `?` everywhere, and it doesn't seem right here, it would make `newTargetBranch !== oldTargetBranch` become `true`.


->  Fix JS error when editing a merged PR's title #30990

Reply:
Actually it is reported from @lunny 
I will close this one.